test/unit/system-info.test.js

```diff
@@ -6,26 +6,407 @@
 'use strict'
 const tap = require('tap')
 const os = require('os')
-const systemInfo = require('../../lib/system-info')
+const proxyquire = require('proxyquire').noPreserveCache()
```

Comment:
Refactoring exposed some lacking test coverage, so I added coverage for the two exported for testing purposes functions `_getProcessStats` and `_getMemoryStats`